src/vs/workbench/contrib/terminal/browser/xterm/decorationAddon.ts

```diff
@@ -28,7 +28,14 @@ const enum DecorationSelector {
 	ErrorColor = 'error',
 	DefaultColor = 'default',
 	Codicon = 'codicon',
-	XtermDecoration = 'xterm-decoration'
+	XtermDecoration = 'xterm-decoration',
+	FirstSplitContainer = '.pane-body.integrated-terminal .terminal-group .monaco-split-view2.horizontal .split-view-view:first-child .xterm'
+}
+
+const enum DecorationStyles {
+	DefaultDimension = 16,
```

Comment:
using 16 here because that's the default codicon size - there is some overlap when it's the second (third, etc) split terminal, in the editor area or side bar. i tried 12 as the default size and it looked too small IMO.
![Screen Shot 2022-02-25 at 10 59 02 AM](https://user-images.githubusercontent.com/29464607/155755953-d7514389-54ce-4e44-be4e-ddd4d56c7274.png)
t

Reply:
LMK what you think